packages/ui/src/views/layouts/PullRequestLayout.tsx

```diff
@@ -21,105 +36,70 @@ enum PullRequestTabsKeys {
   CHANGES = 'changes'
 }
 
-const PullRequestLayout: FC<PullRequestLayoutProps> = ({
+export const PullRequestLayout: FC<PullRequestLayoutProps> = ({
   usePullRequestStore,
   useTranslationStore,
   spaceId,
   repoId,
   updateTitle
 }) => {
-  const { NavLink, Outlet } = useRouterContext()
+  const { Outlet } = useRouterContext()
   const { pullRequest } = usePullRequestStore()
   const { t } = useTranslationStore()
+  const navigate = useNavigate()
+  const location = useLocation()
```

Comment:
You can get `navigate` and `location` from `useRouterContext()`
```suggestion
  const { location, navigate, Outlet } = useRouterContext()
  const { pullRequest } = usePullRequestStore()
  const { t } = useTranslationStore()
```

Reply:
FIxed